hooks/playbooks/barbican-enable-proteccio.yml

```diff
@@ -0,0 +1,90 @@
+---
+- name: Create modified barbican image and get secrets
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  tasks:
+    - name: Check out the role Git repository
+      ansible.builtin.git:
+        dest: "./rhoso_proteccio_hsm"
+        repo: "{{ cifmw_hsm_proteccio_ansible_role_repo | default('https://github.com/openstack-k8s-operators/ansible-role-rhoso-proteccio-hsm.git', true) }}"
+        version: "{{ cifmw_hsm_proteccio_ansible_role_version| default('main', true) }}"
+
+    - name: Create and upload the new Barbican images
+      ansible.builtin.include_role:
+        name: rhoso_proteccio_hsm
+        tasks_from: create_image
+      vars:
+        barbican_src_image_registry: "{{ content_provider_registry_ip }}:5001"
+        barbican_src_image_namespace: "{{ cifmw_update_containers_org | default('podified-antelope-centos9') }}"
+        barbican_src_image_tag: "{{ cifmw_update_containers_tag | default('component-ci-testing') }}"
+        barbican_dest_image_registry: "{{ content_provider_registry_ip }}:5001"
+        barbican_dest_image_namespace: "{{ cifmw_update_containers_org | default('podified-antelope-centos9') }}"
+        barbican_dest_image_tag: "{{ cifmw_update_containers_barbican_custom_tag }}"
+        image_registry_verify_tls: "{{ cifmw_image_registry_verify_tls | default('false', true) }}"
+        proteccio_client_src: "{{ cifmw_hsm_proteccio_client_src }}"
+
+    - name: Create secrets with the HSM certificates and hsm-login credentials
+      ansible.builtin.include_role:
+        name: rhoso_proteccio_hsm
+        tasks_from: create_secrets
+      vars:
+        proteccio_client_name: "{{ cifmw_hsm_client_name }}"
+        proteccio_rc_src: "{{ cifmw_hsm_proteccio_rc_src }}"
```

Comment:
`proteccio_rc_src` is not a role variable either.  This should be changed to `proteccio_conf_src`: https://github.com/openstack-k8s-operators/ansible-role-rhoso-proteccio-hsm?tab=readme-ov-file#secret-generation-variables